Dockerfile

```
@@ -0,0 +1,33 @@
+#ARG BCI_IMAGE=registry.suse.com/bci/bci-busybox
+ARG GO_IMAGE=rancher/hardened-build-base:v1.20.14b1
+
+FROM ${BCI_IMAGE} as bci
+FROM ${GO_IMAGE} as base
+
+RUN set -x && \
+    apk --no-cache add \
+    git \
+    make
+
+FROM base as builder
+ARG ARCH="amd64"
+ARG SRC=github.com/kubernetes/autoscaler
+ARG PKG=github.com/kubernetes/autoscaler
+RUN git clone https://${SRC}.git $GOPATH/src/${PKG}
+ARG TAG=addon-resizer-1.8.20
+ARG ARCH
+WORKDIR $GOPATH/src/${PKG}/addon-resizer
+RUN git branch -a
+RUN git checkout ${TAG} -b ${TAG}
```

```suggestion
ARG TAG="1.8.20"
ARG ARCH
WORKDIR $GOPATH/src/${PKG}/addon-resizer
RUN git branch -a
RUN git checkout addon-resizer-${TAG} -b ${TAG}
```

do we really want the name listed twice?
`rancher/hardened-addon-resizer:addon-resizer-1.8.20-build20240405`